.circleci/config.yml

```diff
@@ -234,6 +254,7 @@ jobs:
           username: $DOCKER_USER
           password: $DOCKER_PASS
     steps:
+      - check-for-deployment
```

Comment:
So in theory this command could never evaluate to true, because it would have already evaluated to false in `docker-image-publish-stage` and halted this job.